### DISCUSSION TITLE
Version 3.5.1RC1 - mot de passe en clair lors d'une modification

### DISCUSSION THREAD

**Discussion by acdct** [JeromeDevome/GRR#242] [Q&A] [answered]
Bonjour,

Je ne sais pas s'il s'agit d'un bug ou pas du coup je poste ma demande ici.

Lorsque je modifie manuellement le mot de passe d'un utilisateur, une fois le formulaire envoyé, tous les paramètres se retrouvent dans l'URL en clair, est-ce normal ?

`http://localhost/grr_351/admin/admin_user_modify.php?reg_login=XXX&reg_nom=XX+XXX&reg_prenom=XXX&reg_email=XXX&reg_statut=administrateur&reg_etat=actif&reg_password=MOT_DE_PASSE_EN_CLAIR&reg_password2=MOT_DE_PASSE_EN_CLAIR&valid=yes&user_login=XXX`

**Comment by ynaessens** [comment, marked as answer]
Bonjour,
merci pour cette remarque. 
En effet ce n'est pas terrible. Je regarde pour passer en POST.
Cordialement,
YN

**Reply by acdct** [reply]
Merci beaucoup !

**Reply by ynaessens** [reply]
Bonjour,
je propose une nouvelle RC qui répond à votre souci de confidentialité : https://github.com/JeromeDevome/GRR/releases/tag/v3.5.1RC2
Merci encore pour votre remarque.
Cordialement,
YN

**Reply by acdct** [reply]
Bonjour,
Merci beaucoup !
Je testerai dès que je pourrai.
Est-ce que cette correction pourra aussi être effective sur la 3.4.3b ?
Bonne journée.
Cordialement

**Reply by acdct** [reply]
Je viens de tester, c'est parfait. Les données ont disparues de l'URL.
Je n'ai par contre pas pu tester authentification LDAP car tous mes comptes sont locaux.
Bonne journée !
